Description: avoid unnecessary version symlinks for client app
Author: Ghislain Antony Vaillant <ghisvail@gmail.com>
Forwarded: no 
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/client/CMakeLists.txt
+++ b/src/client/CMakeLists.txt
@@ -45,7 +45,6 @@
 
 target_link_libraries( clFFT-client clFFT ${Boost_LIBRARIES} ${OPENCL_LIBRARIES} ${CMAKE_DL_LIBS} ${RT_LIB} )
 
-set_target_properties( clFFT-client PROPERTIES VERSION ${CLFFT_VERSION} )
 set_target_properties( clFFT-client PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/staging" )
 if( APPLE )
     # properly deal with RPATH on mac
